packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -284,215 +222,6 @@ QUnit.test('`if` helper with inline form: can use truthy param as binding', func
   equal(view.$().text(), 'yes');
 });
```

Comment:
I *think* the spirit of this test is just to test `{{if dynamicCond dynamicTruthy}}`, which should be more or less covered by your new setup. The differece between this and the new `@htmlbars it can omit the falsy argument` is that the second argument is dynamic, which falls into the bucket of https://github.com/emberjs/ember.js/pull/12920/files#r53242487. If we decided that it doesn't matter, we can just delete this. If we think it matters, we should test that in both configurations (and then delete this).